authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/impl/IdServiceImpl.java

```diff
@@ -327,9 +327,15 @@ private void updateVIDstatus(String vid) throws IdAuthenticationBusinessExceptio
 			vid = securityManager.hash(vid);
 			// Assumption : If transactionLimit is null, id is considered as Perpetual VID
 			// If transactionLimit is nonNull, id is considered as Temporary VID
+
 			if (identityRepo.existsById(vid)
 					&& Objects.nonNull(identityRepo.getOne(vid).getTransactionLimit())) {

```

Comment:
Please replace code duplication for `identityRepo.getOne(vid).getTransactionLimit()` by assigning it into one variable.